bindings/python/iota_sdk/types/transaction_options.py

```diff
@@ -17,30 +18,10 @@ class RemainderValueStrategyCustomAddress:
     """Remainder value strategy for custom addresses.
 
     Attributes:
-        address: An address to move the remainder value to.
-        key_index: The address key index.
-        internal: Determines if an address is a public or an internal (change) address.
-        used: Indicates whether an address has been used already.
+        value: An address to move the remainder value to.
```

Comment:
`strategy` missing?

Reply:
Not sure if it should be part of the comment since users don't set it

Reply:
Can add it if you think it should still be there